proposals/0029-cooperative-vector.md

```diff
@@ -15,6 +15,13 @@
 
 # Cooperative Vectors
 
+Cooperative Vectors is the overall name for this feature, though it doesn't appear
```

Comment:
Do we want to add a bit of history here so readers aren't confused by the name change? Something like:
```
Many implementations implement matrix-matrix and matrix-vector operations by allowing threads in a wave to cooperate under the hood while accessing the specialized hardware to achieve peak performance, hence these APIs that expose the acceleration hardware to HLSL users were clubbed under the moniker **Cooperative/Wave**. But since this is an implementation detail below the level of abstraction of HLSL, the namespce **Linear Algebra** was chosen to categorize these set of operations.
```

Reply:
Added.